org.clojure/clojure/1.4.0/clj/clojure.core/cond/notes.md

```diff
@@ -0,0 +1,18 @@
+## Arities
+
+    [& clauses] -> Any
+
+## Docs
+
+Macro on a series of tests followed by exprs. Each test is evaluated
+one at a time, if a test returns logical true, the corresponding expr
+is evaluated and returned without evaluating any more tests or
+exprs.
+
+Most easily understood as a shorthand for flattening chains of deeply
+nested clj::clojure.core/if forms.
+
+## Notes
+
+- `(cond)` returns nil.
+- returns nil if no tests returned true
```

Comment:
It is common to use `:else` as a default condition at the end of the chain of conditions. `:else` isn't a special value here, it works because keywords always evaluate to truthy. (should be reworded, but the idea is good (I think))